liblwgeom/lwgeom_geos.h

```diff
@@ -19,38 +19,31 @@
  **********************************************************************
  *
  * Copyright 2011 Sandro Santilli <strk@kbt.io>
+ * Copyright 2018 Darafei Praliaskouski <me@komzpa.net>
```

Comment:
Maybe inquire on dev list what proper protocol for copyright is? I'm not sure myself, but the file has certainly been edited since 2018.

Reply:
It's strange. A significant part of the changes is @strk changing the copyrights and not updating years, I expect it to be an oversight that will easily be fixed with next commit of them :)

Copyright process is there in https://trac.osgeo.org/postgis/wiki/DevWikiComitGuidelines

Reply:
Thanks, I hadn't seen that link. People have different conceptions of "significant changes," I guess.

Reply:
@Komzpa  I guess what is considered major is up to interpretation.

snippet from guidelines
``
When substantial contributions are added to a file (such as substantial patches) the author/contributor should be added to the list of copyright holders for the file. 
``
@dbaston  Hmm I didn't make you read that guide :(

Reply:
Sorry for badly-written comment. I've obviously seen the commit guidelines, but didn't recall the snippet Regina posted. I suggested a dev list discussion because there are clearly different conceptions of when copyright should be asserted. I thought it was odd that you were claiming copyright for whitespace changes, but maybe that's just me.